games/i.yaml

```
@@ -97,10 +97,10 @@
   development: active
   originals:
   - 'Cube 2: Sauerbraten'
-  repo: https://github.com/inexor-game/code
+  repo: https://github.com/inexorgame/inexor-core
```

```suggestion
  repo: https://github.com/inexorgame/vulkan-renderer
```
old repo is archived; this appears to be the active dev repo